lib/ui/painting.dart

```diff
@@ -1548,6 +1554,33 @@ class Canvas extends NativeFieldWrapperClass2 {
                   Int32List colors,
                   int blendMode,
                   Float32List cullRect) native "Canvas_drawAtlas";
+
+  void drawShadow(Path path, Color color, {
```

Comment:
Needs docs.  Don't forget to cross-link with ShadowFlags.

I wonder if we should simplify this API to match the level of control we give with physical model layer.  The client code is going to toggle between the two and want the same output.  If we give this low level interface, the client will need to know the details of our physical model implementation to match the parameters.  If we use the same interface, then we can side-step that a bit.